tests/test_s3.py

```diff
@@ -1543,6 +1543,164 @@ def test_has_statement_policy(self):
         resources = p.run()
         self.assertEqual(len(resources), 1)
 
+    def test_bucket_replication_policy_remove(self):
+        replicated_from_name = "replication-from-12345"
+        replicated_to_name = "replication-t0-12345"
+
+        self.patch(s3.S3, "executor_factory", MainThreadExecutor)
+        self.patch(
+            s3.SetBucketReplicationConfig, "executor_factory", MainThreadExecutor
+        )
+
+        # Remove GetBucketLocation from augments
+        # was causing AttributeError: 'FakeHttpResponse' object has no attribute 'content'
+        augments = list(s3.S3_AUGMENT_TABLE)
```

Comment:
every augment should be removed that your not actually using, else for every bucket in this account, there are a dozen api calls, that have greatly expanded the diff in this latest revision compared to previous. if you want to just use source: config as previous alluded you can grep for other examples, else keep the augment table patching to minimize this to only the augments of interest, ie. bucket replication.

Reply:
4b8291831aae806235731e76a1928dbfb915723a absolutely agree! I'll keep going this route for this one but will look into config on the next. I have plans to submit a PR for Block Public Access on S3 buckets tomorrow. Just got to work through a few things on the filter.